cmake/CMakeBuilder.py

```diff
@@ -0,0 +1,13 @@
+from sipbuild import SetuptoolsBuilder
+
+
+class CMakeBuilder(SetuptoolsBuilder):
+    def __init__(self, project, **kwargs):
+        print("Using the CMake builder")
+        super().__init__(project, **kwargs)
+
+    def build(self):
+        """ Only Generate the source files """
```

Comment:
Also same remark as the libsavitar one. You can probably copy the documentation from there.